mnefun/_mnefun.py

```diff
@@ -1002,8 +1004,12 @@ def run_sss_locally(p, subjects, run_indices):
     mne.preprocessing.maxwell_filter
     """
     data_dir = op.join(op.dirname(__file__), 'data')
-    cal_file = op.join(data_dir, 'sss_cal.dat')
-    ct_file = op.join(data_dir, 'ct_sparse.fif')
+    if p.cal_file and p.ct_file is None:
```

Comment:
I think you mean `if p.cal_file is None and p.ct_file is None` but really they should be separate conditionals